src/Endpoints/Delegates/HandlesKeys.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MeiliSearch\Endpoints\Delegates;
+
+use MeiliSearch\Endpoints\Keys;
+
+/**
+ * @property Keys index
+ */
+trait HandlesKeys
+{
+    public function getKeys(): array
+    {
+        return $this->keys->show();
```

Comment:
where is `show` defined in your code? Should it not be in the Keys endpoint ?
Also for consistency why not `all`?

Reply:
I hesitated this function is on the global `Endpoint` file in `src/Contract/Endpoint`. I think you are right I going to rewrite it with all